libs/core/functional/include/hpx/functional/invoke_fused.hpp

```diff
@@ -80,7 +81,7 @@ namespace hpx { namespace util {
     ///
     /// \note This function is similar to `std::apply` (C++17)
     template <typename F, typename Tuple>
-    constexpr HPX_HOST_DEVICE
+    constexpr HPX_HOST_DEVICE HPX_FORCEINLINE
```

Comment:
Do we have any evidence that using forceinline helps here (actually, in general)?

Reply:
I do have evidence from looking at performance results in vtune showing that (at least for MSVC) this option helps reducing the stack noise.

Reply:
Ok, thanks, I was thinking in terms of performance but this is another benefit.